uplink-sys/Cargo.toml

```diff
@@ -10,6 +10,10 @@ license = "Apache-2.0"
 repository = "https://github.com/storj-thirdparty/uplink-rust"
 keywords = ["storj", "storage"]
 
+# Contains relevant information of the Uplink c-binding.
+[package.metadata.uplink-c]
+version = "1.5.0"
```

Comment:
This is manually tracked right?

Maybe add a comment that it should be kept in sync manually with the git submodule.

Reply:
yes, it's. I will add it.